core/raftlease/fsm.go

```diff
@@ -376,15 +379,39 @@ func (r *response) Error() error {
 }
 
 // Notify is part of FSMResponse.
-func (r *response) Notify(target NotifyTarget) {
+func (r *response) Notify(target NotifyTarget) error {
 	// This response is either for a claim (in which case claimer will be set)
 	// or a set-time (so it will have zero or more expirations).
+	var errs []error
 	if r.claimer != "" {
-		target.Claimed(r.claimed, r.claimer)
+		if err := target.Claimed(r.claimed, r.claimer); err != nil {
+			errs = append(errs, errors.Annotatef(err, "unable to claim lease"))
```

Comment:
We should annotate consistently with present tense verbs. We always know that we were "unable" to do something by virtue of the exception. It's just noise in a trace. This would then be "claiming lease".